topics/basics/prepared_statements/README.md

```diff
@@ -1,11 +1,11 @@
 # Prepared Statements
 
-Prepared statements can be used to improve the performance of frequently executed queries. Preparing the query caches it on the Cassandra cluster and only needs to be performed once. Once created, prepared statements should be reused with different bind variables. Prepared queries use the `?` marked to denote bind variables in the query string.
+Prepared statements can be used to improve the performance of frequently executed queries. Preparing the query caches it on the Cassandra cluster and only needs to be performed once. Once created, prepared statements should be reused with different bind variables. Prepared queries use the `?` marker to denote "anonymous" bind variables in the query string. You can also use named bind variables if you specify them as `:name`.
 
 ```c
 /* Prepare the statement on the Cassandra cluster */
 CassFuture* prepare_future
-  = cass_session_prepare(session, "INSERT INTO example (key, value) VALUES (?, ?)");
+  = cass_session_prepare(session, "INSERT INTO example (key, value) VALUES (:key, :value)");
```

Comment:
Why this change? The values of the names can be determined by the `(key, value)` part of the query.

Reply:
ok, will rollback - thought that it will be better visible...